source/lib/storage.js

```diff
@@ -74,3 +75,25 @@ export function setFavoriteBuildings(buildings: string[]) {
 export function getFavoriteBuildings(): Promise<Array<string>> {
 	return getItemAsArray(favoriteBuildingsKey)
 }
+
+/// MARK: Sis
+const courseDataKey = 'sis:course-data'
+export function setTermCourseData(term: number, courseData: Array<CourseType>) {
+	const key = courseDataKey + `:${term}:courses`
+	return setItem(key, courseData)
+}
+export function getTermCourseData(term: number): Promise<Array<CourseType>> {
+	const key = courseDataKey + `:${term}:courses`
+	// return getItemAsArray(key)
+	return AsyncStorage.getItem(`aao:${key}`).then(stored => JSON.parse(stored))
```

Comment:
Please use `getItemAsArray` here, making sure to leave off the `aao:` prefix when you call it.